lib/sda/node.go

```diff
@@ -255,6 +268,13 @@ func (n *Node) Shutdown() error {
 	return nil
 }
 
+// Close shuts down the go-routine and calls the protocolInstance-shutdown
+func (n *Node) Close() error {
+	dbg.Lvl3("Closing node", n.Myself())
```

Comment:
`Name()` is still fine even with this additional info. Otherwise, what do you think about `Info()` ?